heron/metricsmgr/src/java/com/twitter/heron/metricsmgr/sink/WebSink.java

```diff
@@ -84,6 +84,9 @@
   private boolean isFlatMetrics = true;
   private boolean includeTopologyName = true;
   private String topologyName;
+  private long cacheMaxSize;
```

Comment:
I would be just repeating this comment,

https://github.com/tc-dc/heron/blob/ed0e25cb71dd882c159ef4bbcf1fe634a891667c/heron/metricsmgr/src/java/com/twitter/heron/metricsmgr/sink/WebSink.java#L74